packages/cloud-cognitive/src/components/Datagrid/Datagrid/addons/CustomizeColumns/CustomizeColumnsModal.js

```diff
@@ -76,12 +81,30 @@ const CustomizeColumnsModal = ({
     }
   };
 
+  const getVisibleColumnsCount = useCallback(() => {
+    let result = 0;
+    for (let i = 0; i < columnObjects.length; i++) {
+      if (columnObjects[i].isVisible == true) {
+        result++;
+      }
+    }
```

Comment:
@kai-liu 

Probably another opportunity to use array functions / forEach loop.